### PR TITLE
Change for...<length to for...of loop

### PR DESCRIPTION
**Pull request by gannerr**
Change for...<length to for...of loop

- Improves readability of output of reviews.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -21,8 +21,8 @@ function expandImage(imgs) {
 
 function getReviews() {
   fetch('/data').then(response => response.json()).then((reviews) => {
-    for(var i = 0; i < reviews.length; i++) {
-      document.getElementById('reviews-container').innerText += reviews[i] + '\n';
+    for(var review of reviews) {
```
Comment (avaleske) [inline portfolio/src/main/webapp/script.js:24]: nbd but I think `let` is slightly more correct here, instead of var.

Comment (gannerr) [reply portfolio/src/main/webapp/script.js:24]: very true. I will change

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -41,4 +40,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
+    // Get the input from the form.
+    String name = getParameter(request, "reviewer-name", "");
+    String input = getParameter(request, "reviewer-input", "");
+    String review = name + " said: \n" + input;
+    reviews.add(review);
+    response.sendRedirect("/index.html");
+  }
+
+  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
+    String value = request.getParameter(name);
```
Comment (avaleske) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:55]: Too bad getParameter() doesn't return an Optional<>...

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -34,6 +34,17 @@
 
       <button onclick="getReviews()">Gimme some reviews!</button>
       <div id="reviews-container"></div> 
+
+      <form action="/reviews" method="POST">
+
```
Comment (avaleske) [inline portfolio/src/main/webapp/index.html:39]: No need for the empty line

**Pull request merged**